src/leaner-record/index.ts

```diff
@@ -0,0 +1,31 @@
+import {OauthRestService} from '../lib/http/oauthRestService'
+import {LearnerRecordConfig} from './learnerRecordConfig'
+import {Auth} from '../identity/auth'
+import {EventRecordFactory} from './model/factory/eventRecordFactory'
+
+export class LearnerRecord {
+	private _restService: OauthRestService
+	private _eventRecordFactory: EventRecordFactory
+
+	constructor(config: LearnerRecordConfig, auth: Auth, eventRecordFactory: EventRecordFactory) {
+		this._restService = new OauthRestService(config, auth)
+
+		this._eventRecordFactory = eventRecordFactory
+	}
+
+	async getEventRecord(eventId: string) {
+		const data = await this._restService.get('/events/' + eventId)
```

Comment:
For consistency sake, could we use string interpolation:

`const data = await this._restService.get(`/events/${eventId}`)`